script/run-cypress-tests-circle.sh

```diff
@@ -4,8 +4,10 @@ if [ "$(find src -name '*.cypress.spec.js' | wc -l)" -eq 0 ]; then
   echo "No Cypress tests found."
   exit 0
 else
-  # Flag to indicate to tests that they are running in a CI environment.
+  # Flags to indicate to tests that they are running in a CI environment
+  # and also that they are running in CircleCI specifically.
   export CYPRESS_CI=$CI
+  export CYPRESS_CIRCLECI=$CIRCLECI
```

Comment:
Is `CYPRESS_CIRCLECI` used anywhere, or is this just being created proactively?

Reply:
I *think* it's being used in the file below, line 39.

Reply:
Good call. 

>Any OS-level environment variable on your machine that starts with either `CYPRESS_` or `cypress_` will automatically be added to Cypress’ environment variables and made available to you. - https://docs.cypress.io/guides/guides/environment-variables.html#Option-3-CYPRESS